doc/specs/vulkan/chapters/initialization.txt

```diff
@@ -30,41 +30,35 @@ Loaders are encouraged to export function symbols for all other core
 Vulkan commands as well; if this is done, then applications that use only
 the core Vulkan commands have no need to use fname:vkGetInstanceProcAddr.
 
-Function pointers to commands that don't operate on a specific instance can:
-be obtained by using this command with pname:instance equal to `NULL`. The
-following commands can: be accessed this way:
-
-  * flink:vkEnumerateInstanceExtensionProperties
-  * flink:vkEnumerateInstanceLayerProperties
-  * flink:vkCreateInstance
-
-If pname:instance is a valid sname:VkInstance, function pointers to any
-commands that operate on pname:instance or a child of pname:instance can: be
-obtained. The returned function pointer must: only be called with a
-dispatchable object (the first parameter) that is a child of pname:instance.
-
-If pname:pName is not the name of a core Vulkan command, or is an
-extension command for any extension not supported by any available layer or
-implementation, then fname:vkGetInstanceProcAddr will return `NULL`.
-
-ifdef::editing-notes[]
-[NOTE]
-.editing-note
-====
-(Jon, Bug 14886 / Gitlab issue 4) The WSI group tentatively agreed that the
-WSI extensions were special, and should get static entry points in link
-libraries and prototypes in +vulkan.h+, while future extensions would have
-to be dynamically loaded. If this decision is upheld by the group as a
-whole, it would probably be encoded in the previous paragraph, in the WSI
-extensions branch of the Specification.
-
-However, this decision hasn't been fully signed off on by the entire Vulkan
-WG yet AFAIK. Note that implementations typically won't support many of the
-WSI extensions, so ``static entry points'' doesn't relieve apps of the
-neccessity of runtime enabling and testing of each extension before using
-it.
-====
-endif::editing-notes[]
+The table below defines the various use cases for fname:vkGetInstanceProcAddr
+and expected behavior ("fp" is function pointer) for each case. Each rule
+is tested in the order below.
+
+.vkGetInstanceProcAddr behavior
+[width="80%",options="header",align="center"]
+|=====
+| pname:instance | pname:pName | behavior
+| invalid instance | * | undefined
+| * | NULL | undefined
+| NULL | flink:vkEnumerateInstanceExtensionProperties | returns fp
+| NULL | flink:vkEnumerateInstanceLayerProperties | returns fp
+| NULL | flink:vkCreateInstance | returns fp
+| NULL | * (not any row/case above) | returns NULL
+| instance | flink:vkEnumerateInstanceLayerProperties | undefined
+| instance | flink:vkEnumerateInstanceExtensionProperties | undefined
+| instance | flink:vkCreateInstance | undefined
```

Comment:
Okay, it can be NULL. I had checked the code for both the Android and Windows/Linux loader and it looked like the desktop loader did not define this case, but I was mistaken. Both loaders will return NULL when given a non-NULL VkInstance and one of the three global functions.